scalus-core/shared/src/main/scala/scalus/builtin/ByteString.scala

```diff
@@ -183,4 +186,8 @@ object ByteString extends ByteStringOffchainApi {
         /** Prepends a BigInt to a ByteString and returns a new ByteString */
         inline infix def +:(bs: ByteString): ByteString = Builtins.consByteString(b, bs)
 
+    extension (sc: StringContext)
+        inline def b(args: Any*): ByteString =
```

Comment:
I like the idea but I'm not sure about `b` prefix. Maybe `utf8` or `bs`. Let's do `utf8`